### ISSUE TITLE
JAWS ignores aria-labelledby on TH

### ISSUE THREAD

**Issue by wkeese** [FreedomScientific/VFO-standards-support#909]
# Summary

Navigate to https://stackblitz.com/edit/stackblitz-starters-bsnuekeb?file=index.html or run locally.

Tab to the two buttons in the table's `<tbody>` and listen to the announcements of the column names.

# Expected results

JAWS should announce the column names as "banana 1" and "banana 2" per the   `<th>` elements' `aria-label` and `aria-labelledby` attributes.

# Actual results

JAWS obeys the `aria-label` but ignores the `aria-labelledby`, thus announcing the second column name as "tomato 2".

# Additional Information
## JAWS version and build number
JAWS 2025

## Operating System and version
Windows 11

## Browser and version:
Chrome 137

**Comment by stevefaulkner** [comment]
test case codepen: https://codepen.io/stevef/pen/raVJJeK

can reproduce, note correct acc names are exposed for both header cells in browser acc tree.
Also note NVDA does not appear to announce either as _Banana_

**Comment by aardrian** [comment]
How would you propose this interact with the [`abbr` attribute](https://html.spec.whatwg.org/multipage/tables.html#attr-th-abbr)? ([current AAM mapping](https://w3c.github.io/html-aam/#att-abbr))

**Comment by JeremiahRogersVispero** [comment]
This is related to #884  and is in the bug system.

**Comment by wkeese** [comment]
> How would you propose this interact with the [`abbr` attribute](https://html.spec.whatwg.org/multipage/tables.html#attr-th-abbr)? ([current AAM mapping](https://w3c.github.io/html-aam/#att-abbr))

Oh, I didn't know about that attribute!  I think `abbr` should take precedence over `aria-label` (when announcing the column because the user is navigating table data cells).

Too bad neither `aria-label` nor `abbr` [work on NVDA](https://github.com/nvaccess/nvda/issues/3566) but that's not a problem for the tracker :-).
 


> This is related to [#884](https://github.com/FreedomScientific/standards-support/issues/884) and is in the bug system.

Oh, sorry, I searched for pre-existing tickets but didn't find that one.
